source/clear-linux/network/network-bonding.rst

```diff
@@ -0,0 +1,131 @@
+``/etc/systemd/network`` directory:
+
+.. code-block:: console
+
+   # mkdir -p /etc/systemd/network
+
+In this directory, we'll create the configuration files for the virtual device
+and the network settings for it and the underlying physical interfaces.
+
+Next, we'll configure systemd to create a virtual network device, ``bond1``.
+The syntax for this file is defined in the systemd.netdev_ manpage, and we're
+copying `an example`__ verbatim. Note that ``802.3ad`` mode requires explicit
+support from your NICs and network switch. This and other modes may also
+require additional configuration of your network switch.
+
+__ https://www.freedesktop.org/software/systemd/man/systemd.netdev.html#id-1.20.10
+
+.. code-block:: ini
+   :caption: 30-bond1.netdev
+   :linenos:
+
+   [NetDev]
+   Name=bond1
+   Kind=bond
+
+   [Bond]
+   Mode=802.3ad
+   TransmitHashPolicy=layer3+4
+   MIIMonitorSec=1s
+   LACPTransmitRate=fast
+
+We need to configure the slave interfaces, assigning them to our new
+``bond1``, using the syntax in systemd.network_.
+
+.. code-block:: ini
+   :caption: 30-bond1-enp1s0.network
+   :linenos:
+
+   [Match]
+   Name=enp1s0f*
+
+   [Network]
+   Bond=bond1
+
+   [Link]
+   MTUBytes=9000
+
+.. note::
+
+   I'm bonding all four ports of a quad-port NIC, with names in the range
+   ``enp1s0f0-enp1s0f3``, so I can use a single file with a wildcard match. You
+   may also create a separate file for each NIC, particularly if they have names
+   that are not wildcard-friendly. This configuration assigns each NIC as a slave
+   of ``bond1``. Notice that we do not assign addresses or DHCP support to the
+   individual NICs.
+
+.. note::
+
+   I'm also enabling jumbo frames of up to 9000 bytes to optimize large data
+   transfers on my local network. Again, your NICs and switch must support jumbo
+   frames, and your switch may require additional configuration.
+
+   Finally, we need to define the network configuration for the bonded interface.
+
+.. code-block:: ini
+   :caption: 30-bond1.network
+   :linenos:
+
+   [Match]
+   Name=bond1
+
+   [Network]
+   BindCarrier=enp1s0f0 enp1s0f1 enp1s0f2 enp1s0f3
+   Address=192.168.1.201/24
+
+   [Link]
+   MTUBytes=9000
+
+.. note::
+
+   Since ``bond1`` is a virtual interface, it has no concept of physical link
+   status. The ``BindCarrier`` directive indicates that the link status of this
+   interface is determines by the status of the listed slave devices.
+
+.. note::
+
+   As this is the logical interface we'll be using, we assign it an IP address.
+   DHCP is more complicated with bonded interfaces, and is not covered in this
+   guide.
+
+.. note::
+
+   Once again, I'm enabling jumbo frames of up to 9000 bytes. This option must be
+   enabled for all slave interfaces *and* the bonded interface in order to take
+   effect.
+
+To apply the new network configuration:
+
+.. code-block:: console
+
+   # systemctl restart systemd-networkd
```

Comment:
There is an implicit assumption here that this command should be run as `root` (due to the hash prompt). I think it would be clearer to state at the start of the guide that all commands that use this prompt should be run as `root` (unless there is a "meta-guide" somewhere that already explains this?)